src/pudl/metadata/fields.py

```diff
@@ -1611,6 +1643,18 @@
         "type": "number",
         "description": "Total number of flue gas desulfurization unit scrubber trains.",
     },
+    "former_conformed_name": {
+        "type": "string",
+        "description": "Former name of the company.",
+    },
```

Comment:
By analogy with `company_name` and `company_name_raw` might this be clearer as `company_name_former`? What does "conformed" mean in this context?

Reply:
Yep, `former_confermed_name` comes directly from a field pulled from the 10-K filing. I don't see any reason to keep `conformed` though. We don't use this field in any of the modeling bits.